woocommerce/woocommerce-android
# Added logic to display sale price even if it is zero

Fixes #2099 by adding logic to display sale price even if it is zero. In core, a zero sale price is possible and might have some use cases when a product is free for a period of time.

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/77425663-141dc100-6df9-11ea-9d3e-24ccb5e48b70.png" width="300"/>

#### Testing
- Click on a Product from the product list.
- Click on the Pricing section.
- Change the `sale price` to 0 and click `DONE`.
- Notice the sale price is displayed in the product detail screen.
- Click on `UPDATE` and notice the sale price is still displayed.
- Click on the Pricing section again and this time change the sale price to anything other than 0.
- Click on `DONE` and notice that the sale price is correctly displayed.
- Click on `UPDATE` and notice the sale price is correctly updated.

Update release notes:
